manual/src/refman/exten.etex

```diff
@@ -123,5 +123,30 @@ that are implemented in OCaml, but not described in chapter \ref{c:refman}.
 %HEVEA\cutname{effects.html}
 \input{effects.tex}
 
+\section{s:memory-model}{Memory model for parallel OCaml programs}
+%HEVEA\cutname{lang_memorymodel.html}
+(Introduced in 5.0)
+
+OCaml 5.0 has introduced support for running parallel OCaml code in the same
```

Comment:
Is this text useful? For the reader to see this text, they should already be in the language extension section.

Reply:
dropped